epoxy-adapter/src/main/java/com/airbnb/epoxy/ModelGroupHolder.kt

```diff
@@ -9,12 +9,12 @@ import androidx.recyclerview.widget.RecyclerView
 import com.airbnb.viewmodeladapter.R
 import java.util.ArrayList
 
-class ModelGroupHolder(parent: ViewParent) : EpoxyHolder() {
+class ModelGroupHolder(private val modelGroupParent: ViewParent) : EpoxyHolder() {
```

Comment:
I'd like to follow up later to see if we can avoid storing this parent, as theoretically this holder can get recycled to be used by another recyclerview